spec/features/admin/merchants/index_spec.rb

```diff
@@ -1,11 +1,9 @@
 require 'rails_helper' 
 
-RSpec.describe "Admin Merchants Index", type: :feature do 
+RSpec.describe Merchant, type: :feature do 
```

Comment:
Nice, this is probably good to have as convention more than a description in quotation marks.